Bigtable/src/BigtableClient.php

```diff
@@ -156,39 +157,37 @@ public function instance($instanceId, array $instance = [])
      *
      * Example:
      * ```
-     * $cluster = $bigtable->clusterMetadata('my-cluster', 'us-east-b');
+     * $cluster = $bigtable->clusterMetadata();
      * ```
      * @param string $clusterId The cluster ID
      * @param string $locationId The location ID
      * @param int $storageType The storage media type for persisting Bigtable data. Possible values include
-     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_SSD`,
-     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_HDD` and
-     *            `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
-     *            **Defaults to** `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
-     * @param int $serveNodes
-     *
+     *        `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_SSD`,
+     *        `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_HDD` and
+     *        `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
+     *        **Defaults to** `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_UNSPECIFIED`.
+     * @param int $serveNodes The number of nodes allocated to this cluster.
+     *        More nodes enable higher throughput and more consistent performance.
+     * }
      * @return array
      */
     public function clusterMetadata(
```

Comment:
I think this is the worse version than before.
What's the point of creating a metadata as an empty array?
And why do you change the default value of storage type to null? This doesn't make sense based on the current PHPDoc too.